_data/accessibility-tests/summary-box.yml

```diff
@@ -0,0 +1,61 @@
+title: Summary Box
+component_status: pass
+test_items:
+  # General tests
+  - summary: Summary box headings are in the correct heading structure of the page.
+    summary_additional: When you navigate to a summary box, you can see (and hear with a screen reader) that headings follow proper heading structure within the page.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.1
+  - summary: Summary box background and text meet contrast requirements.
+    summary_additional: When you use a color contrast checker to view the summary box, the text and background within it have at least a 4.5:1 contrast.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.3
+  - summary: Links provide direct location description.
+    summary_additional: When you read the text of any link in the summary box, you can understand where the link will take you.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.4
+  - summary: All headings are clear and descriptive.
+    summary_additional: When you view the summary box, the headings are clear and concise.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.6
```

Comment:
**Question:** We're listing `2.4.6` in General & Screen Reader tests. Do we need both? Do we list it twice in other tests?

I'm thinking we should compare to other components that list `2.4.6` and pick the one that's more common/logical for this component

Reply:
If I understand what you mean, accordion and table both list 2.4.6 more than once

Reply:
Updated in [7f309b3](https://github.com/uswds/uswds-site/pull/3016/commits/7f309b3a5923de678481e29da05bc1f692a628df) to remove screen reader test in favor of the general one